### DISCUSSION TITLE
C objective sdk for 1DS

### DISCUSSION THREAD

**Discussion by swasti29** [microsoft/cpp_client_telemetry#1064] [Q&A]
I have found the c objective sdk for 1DS here: https://eng.ms/docs/products/one-observability-microsoft/june18-ios-getting_started. Which one is recommended to use for C objective client applications?

**Comment by lalitb** [comment]
Please refer to https://github.com/microsoft/cpp_client_telemetry for all latest documents. So for iOS : https://github.com/microsoft/cpp_client_telemetry/blob/main/docs/cpp-start-ios.md
